resources/public/api/conf/2.0/schemas/income-sa-additional-information-response.json

```diff
@@ -48,6 +48,11 @@
                 "items": {
                   "type": "object",
                   "properties": {
+                    "utr": {
+                      "type": "string",
```

Comment:
UTR has a pattern assigned to it in the schemas :- '^[0-9]{10}$' , can you add this to the schemas please.